packages/compass-import-export/src/components/import-preview.tsx

```diff
@@ -322,10 +318,6 @@ function FieldTypeHeading({
   onFieldCheckedChanged: (fieldPath: string, checked: boolean) => void;
   setFieldType: (fieldPath: string, fieldType: string) => void;
 }) {
-  if (field.type === 'placeholder') {
-    return null;
-  }
-
   const children = (
```

Comment:
Should we cap the length of strings shown? Is it a big performance hit if there are some huge arrays or strings?

Reply:
The arrays become strings and the table already turns them into ellipses. Seems fine? We could always already have giant description fields in there.

Reply:
I'm capping the string length now. Will push the changes..